jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/PdfModule.java

```diff
@@ -1361,9 +1361,16 @@ protected boolean parseTrailer(RepInfo info, boolean prevOnly)
 				throw new PdfInvalidException(MessageConstants.PDF_HUL_75, // PDF-HUL-75
 						_parser.getOffset());
 			}
-			_encryptDictRef = (PdfIndirectObj) _trailerDict
-					.get(DICT_KEY_ENCRYPT); // This is at least v. 1.1
-			_encrypted = (_encryptDictRef != null);
+			PdfObject encryptObj =  _trailerDict.get(DICT_KEY_ENCRYPT); 
```

Comment:
This is a solid approach, but the if/else leaves the possibility that the instance returned here isn't "tested" at all. Put another way, what happens when `encryptObj` ISN'T an instance of `PdfIndirectObj` OR `PdfDictionary`? This may be an error that gets picked up elsewhere, but I wonder if there shouldn't be a default condition here that marks an error?